spec/controllers/ctc/portal/pages_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+describe Ctc::Portal::PagesController do
+  let(:primary_birth_date) { 30.years.ago }
+  let(:wages_amount) { 1000 }
+  let(:intake) do
+    create :ctc_intake,
+           primary_birth_date: primary_birth_date,
+           current_step: "en/portal/dependents/not-eligible",
+           dependents: [],
+           claim_eitc: 'yes',
+           primary_tin_type: 'ssn',
+           exceeded_investment_income_limit: 'no'
+
+  end
+  let!(:w2) { create :w2, wages_amount: wages_amount, intake: intake }
+  let!(:client) { create :client, intake: intake, tax_returns: [create(:tax_return)] }
+
+  before do
+    allow(subject).to receive(:open_for_eitc_intake?).and_return true
+    sign_in intake.client
+  end
+
+  context "#no_eligible_dependents" do
+    it "renders no_eligible_dependents template" do
+      get :no_eligible_dependents
+      expect(response).to render_template :no_eligible_dependents
```

Comment:
Rendering a template with the same name as the controller action is something we generally trust Rails to do. Given that, I'd say it's better to check that the response is OK in this test. A redirect is a 302/301, not 200 (OK), after all.